File: python/26.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l = []
        for i in nums:
            if i not in l:
                l.append(i)
        nums.clear()
        for i in l:
            nums.append(i)
        return len(nums)

File: rust/26.rs
impl Solution {
    pub fn remove_duplicates(nums: &mut Vec<i32>) -> i32 {
        let mut l: Vec<_> = vec!();
        for i in nums.into_iter() {
            if !(l.contains(i)) {
                l.push(*i as i32);
            }
        }
        nums.clear();
        for i in l {
            nums.push(i);
        }
        nums.len() as i32
    }
}
